torchci/pages/api/usage-log-aggregator/lambda.ts

```diff
@@ -0,0 +1,14 @@
+import * as restc from "typed-rest-client/RestClient";
+import { NextApiRequest, NextApiResponse } from "next";
+
+// TODO: Create a hostname and secure this API
```

Comment:
is this still a todo? How important is it?

Reply:
Oh, I have a comment for this in https://github.com/pytorch/test-infra/pull/858.  I accidentally have some of the API part included in this PR.  It would be great if you can help take a look at https://github.com/pytorch/test-infra/pull/858. I split this up to help with the reviewing.

Back to the question: None of the API we are using in OSS CI have an API key I think including the one that accepts GitHub hooks and write to S3. This API only reads the data from S3 (read-only), so I think we can leave the API key part later